Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: airflow
-  repository: https://github.com/astronomer/airflow/releases/download/oss-helm-chart/1.11.1-astro
-  version: 1.11.1-astro
-digest: sha256:6d7c55a8e643d3e50723187c70177ec8baa8610a4b27f430a28805afaaf7a05f
-generated: "2023-12-06T11:30:00.49718-05:00"
+  repository: https://airflow.apache.org
+  version: 1.11.0
```

Comment:
shouldn't this be newer? didn't we make our own 1.11.1-astro because we were waiting for something in OSS 1.11.1 ?